addons/website_sale/controllers/main.py

```diff
@@ -228,6 +236,7 @@ def shop(self, page=0, category=None, search='', **post):
             'attributes': attributes,
             'compute_currency': compute_currency,
             'keep': keep,
+            'parent_category': recursive_category(category) if category else [],
```

Comment:
Note: is it really necessary ? As you browse childs of categories, no need to have all ancestors of a current category. To check.